lemarche/www/pages/urls.py

```diff
@@ -160,6 +161,11 @@
     path("track/", TrackView.as_view(), name="track_frontend"),
     # Calculator endpoints
     path("calibrer-achat-socialement-responsable/", ImpactCalculatorView.as_view(), name="impact_calculator"),
+    path(
+        "calculez-limpact-social-dun-achat-inclusif/",
```

Comment:
```suggestion
        "calculer-impact-social-achat-inclusif/",
```
plus simple + verbe comme `calibrer` ?